pkg/server/container_create.go

```diff
@@ -589,7 +594,13 @@ func generateApparmorSpecOpts(apparmorProf string, privileged, apparmorEnabled b
 	}
 }
 
-// generateUserString generates valid user string based on OCI Image Spec v1.0.0.
+// generateUserString generates valid user string based on OCI Image Spec
+// v1.0.0.
+//
+// CRI defines that the following combinations are valid:
+//
+// uid, uid/gid, username, username/gid
```

Comment:
`/` -> `:`?

Reply:
I can see why this is confusing. I changed the comment.

Reply:
Ha, I was confused... Sorry. :P